packages/main/src/plugin/extension/extension-development-folders.ts

```diff
@@ -147,4 +156,13 @@ export class ExtensionDevelopmentFolders {
   getDevelopmentFolders(): ExtensionDevelopmentFolderInfo[] {
     return this.#developmentFolders.map(path => ({ path }));
   }
+
+  addExternalExtensionId(extensionId: string): void {
```

Comment:
question: what would happen if I have already the extension installed (let's say from the catalog), then try to start in dev mode?